components/Repositories/YoloChart.tsx

```diff
@@ -0,0 +1,150 @@
+import * as HoverCard from "@radix-ui/react-hover-card";
+import Link from "next/link";
+import { FaArrowLeft, FaRegHandPeace } from "react-icons/fa6";
+import Skeleton from "react-loading-skeleton";
+import SkeletonWrapper from "components/atoms/SkeletonLoader/skeleton-wrapper";
+import Card from "components/atoms/Card/card";
+import Avatar from "components/atoms/Avatar/avatar";
+import HoverCardWrapper from "components/molecules/HoverCardWrapper/hover-card-wrapper";
+import { getAvatarByUsername } from "lib/utils/github";
+import Button from "components/shared/Button/button";
+
+type YoloChartProps = {
+  yoloStats: RepositoryYolo | undefined;
+  uniqueYoloCoders: Set<string> | undefined;
+  repository: string;
+  isLoading: boolean;
+  range: number;
+  backButtonOnClick?: () => void;
+  showHoverCards?: boolean;
+  className?: string;
+};
+
+export default function YoloChart({
+  yoloStats,
+  uniqueYoloCoders,
+  repository,
+  isLoading,
+  range,
+  backButtonOnClick,
+  showHoverCards,
+  className,
+}: YoloChartProps) {
+  return (
+    <Card className={`${className ?? ""} flex flex-col gap-4 w-full h-full items-center`}>
+      <div className="flex flex-col gap-2 items-start w-full">
+        {backButtonOnClick && (
+          <Button variant="text" onClick={backButtonOnClick} className="flex gap-2 text-sauced-orange">
+            <FaArrowLeft /> Back
+          </Button>
+        )}
+        <section className="flex flex-col lg:flex-row w-full items-start lg:items-center gap-4 lg:justify-between px-4">
+          <header className="flex w-full justify-between items-center">
+            <div className="flex gap-2 items-center">
+              <FaRegHandPeace className="text-xl" />
+              <h3 className="text-sm font-semibold xl:text-lg text-slate-800">YOLO Coders</h3>
+            </div>
+          </header>
+        </section>
+      </div>
+
+      <section className="w-full px-4 flex flex-col gap-2 text-xs xl:text-sm">
+        {isLoading ? (
+          <Skeleton height={32} />
+        ) : (
+          <p className="text-slate-500">
+            {uniqueYoloCoders && (
+              <>
+                <span className="font-semibold text-black">{`${uniqueYoloCoders.size} `}</span> contributor
+                {uniqueYoloCoders.size > 1 && "s"} {uniqueYoloCoders.size > 1 ? "have" : "has"} pushed{" "}
+                <span className="font-semibold text-black">{yoloStats?.num_yolo_pushed_commits} commits </span>
+                directly to the main branch in the last <span className="font-semibold text-black">{range}</span> days.
```

Comment:
nit: if the value is 0 we should say something like "No one has...."

Reply:
If there is 0, the banner in the `LotteryFactorChart` wouldn't show and there wouldn't be any way to access the `YoloChart`, unless you want this to render no matter what like for StarSearch?

Reply:
I'd rather just not show the banner, to simplify the UI

Reply:
I only brought it to because it showed "0 contributor..." when I was testing it.